### ISSUE TITLE
Resident Evil 2 Remake Profile No Longer Working in 2021

### ISSUE THREAD

**Issue by F0XTHREE** [antonpup/Aurora#2275]
### Expected Behaviour
 Lighting working with the health/poison status in game.

### Actual Behaviour
 Lighting staying stuck in the "offgame" or "dead" layers.

### Reproduction steps
Using the profile while in game.

### Frequency
100% of the time.

### Aurora Version:
0.8.1

### Previously an Issue?
 In 2019 the profile worked fine.

**Comment by F0XTHREE** [comment]
[2021-02-13 18.06.55.log](https://github.com/antonpup/Aurora/files/5980417/2021-02-13.18.06.55.log)

**Comment by angelaly11** [comment]
yeah i had similar issues last year when i tried to use the resident evil 2 remake profile in aurora, tbh it was really frustrating cuz the lighting just wouldn't work right and would stay stuck on "offgame" or "dead". 

anyway, check out this post https://gfdgdsfgdfsgsdf.cyou/pages/81fdf9d2fdce4e79a1fa26ac63d38c52 it helped me sort out the issue back then and i think you'll find what you need there.
